src/Aquifer.Data/Entities/TranslationPairEntity.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Aquifer.Data.Entities;
+
+[EntityTypeConfiguration(typeof(TranslationPairEntityConfiguration))]
+public class TranslationPairEntity
+{
+    public int Id { get; set; }
+    public required LanguageEntity Language { get; set; }
+    public required int LanguageId { get; set; }
+    public required string Key { get; set; }
+    public required string Value { get; set; }
```

Comment:
I know this was on the Linear ticket, but these names feel kind of generic. Would something like `SourcePhrase` and `TargetPhrase` make more sense?

Reply:
I think when it's paired with the entity name `TranslationPair.Key` and `TranslationPair.Value`, it makes a lot of sense.